src/components/App/App.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
```

Comment:
Not important for this one, but for future PRs, let's keep the name of the files using kebabCase (all lowercase and using `-` to separate words), just for consistency with other projects

Reply:
Okay, thanks for comment. I already changed naming in my local project. Will update it in next PR

Reply:
yep yep, no prob 👍